src/chocolatey/infrastructure.app/nuget/NugetList.cs

```diff
@@ -48,12 +64,11 @@ public static IEnumerable<IPackage> GetPackages(ChocolateyConfiguration configur
                 results = results.Skip(configuration.ListCommand.PageSize * configuration.ListCommand.Page.Value).Take(configuration.ListCommand.PageSize);
             }
 
-            return results.OrderBy(p => p.Id)
-                        .AsEnumerable()
-                        .Where(PackageExtensions.IsListed)
-                        .Where(p => configuration.Prerelease || p.IsReleaseVersion())

```

Comment:
Both the above checks were completely superfluous new as I can tell. The former was redundant with checks the server is already doing, and the later is redundant with the IsPrelease check.